README.md

```diff
@@ -144,7 +144,7 @@ From the KRACK <a href="https://www.krackattacks.com/">website</a>:
 | pfSense | [LINK](https://redmine.pfsense.org/issues/7951) | N/A | 2017-10-17 | 2017-10-17 |  |
 | Pioneer | No Known Official Response | N/A | 2017-10-17 | 2017-10-17 |  |
 | Qualcomm Atheros | No Known Official Response | N/A | 2017-10-16 | 2017-10-16 |  |
-| Rachio | No Known Official Response | N/A | 2017-10-17 | 2017-10-17 |  |
+| Rachio | No Known Official Response | Support response: "When it boils down into it, the KRACK attack can only target improperly done HTTPS / SSL connections, and we are perfectly safe in that regard. There is no need for our controller to get an update due to the leak itself, due to the massive lack of a GUI there is nothing at risk from our controller. <br/> From what I can see in my research and testing, KRACK vulnerability cannot potentially modify data on the network, or even eavesdrop from our controller. <br/> The absolute only thing at risk, after thorough testing, that a KRACK attacker would be able to *potentially* see is that you have a Rachio on your network. And even then, the only way they have the slightest ability to get any further would be via timing analysis, and even then it only would be your watering times.  | 2017-10-17 | 2017-10-17 |  |
```

Comment:
Why are they not issuing a patch? It might not affect their upper application protocols, but it would seem like a good idea to patch all vulnerabilities.

Reply:
I can't answer why they won't issue a patch. The responses are twitter, nothing official on their sites.

This is ecobee: https://twitter.com/ecobee/status/920316685193707521 and https://twitter.com/ecobee/status/920316741334552576

This is Rachio: https://twitter.com/Fezmid/status/920261178852630530

I expect we'll see a lot of these kind of "we use https, so we won't patch" responses from IoT vendors. Hence the industry-wide griping about IoT security for the past few years.

Reply:
This is horrible, but perhaps expected. I will issue the merge here soon.